Makefile

```diff
@@ -0,0 +1,123 @@
+PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
```

Comment:
Do we need this parameters?
`PACKAGES_NOSIMULATION`
`PACKAGE_SIMTEST`

Reply:
No, they are leftovers from a previous version I was working on. Removed.